Invoking command in '/home/ubuntu/crawler_ws/build/gps_interface': PYTHONPATH=/home/ubuntu/crawler_ws/install/gps_interface/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py develop --prefix /home/ubuntu/crawler_ws/install/gps_interface --uninstall --editable --build-directory /home/ubuntu/crawler_ws/build/gps_interface/build
Invoked command in '/home/ubuntu/crawler_ws/build/gps_interface' returned '0': PYTHONPATH=/home/ubuntu/crawler_ws/install/gps_interface/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py develop --prefix /home/ubuntu/crawler_ws/install/gps_interface --uninstall --editable --build-directory /home/ubuntu/crawler_ws/build/gps_interface/build
Invoking command in '/home/ubuntu/crawler_ws/src/gps_interface': PYTHONPATH=/home/ubuntu/crawler_ws/install/gps_interface/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/gps_interface build --build-base /home/ubuntu/crawler_ws/build/gps_interface/build install --prefix /home/ubuntu/crawler_ws/install/gps_interface --record /home/ubuntu/crawler_ws/build/gps_interface/install.log --single-version-externally-managed
Invoked command in '/home/ubuntu/crawler_ws/src/gps_interface' returned '0': PYTHONPATH=/home/ubuntu/crawler_ws/install/gps_interface/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/gps_interface build --build-base /home/ubuntu/crawler_ws/build/gps_interface/build install --prefix /home/ubuntu/crawler_ws/install/gps_interface --record /home/ubuntu/crawler_ws/build/gps_interface/install.log --single-version-externally-managed
